chia/wallet/puzzles/vault_p2_recovery.clsp

```diff
@@ -47,20 +47,23 @@
     )
   )
 
-  (list
-    (list AGG_SIG_ME BLS_PK (sha256tree recovery_conditions))
-    (list CREATE_COIN
-      (create_recovery_puzzlehash
-        P2_1_OF_N_MOD_HASH
-        FINISH_RECOVERY_MOD_HASH
-        P2_SECP_PUZZLEHASH
-        TIMELOCK
-        recovery_conditions
+  (let
```

Comment:
I'm double checking with Art but I'm 80% sure that `assign` is more appropriate here so you don't end up calculating the puzzle hash twice.